src/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+const path = require('path');
+
+const app = express();
+const port = 3002;
+
+// The list of templates with Cypress tests
+const templates = ['verify'];
+
+templates.forEach((template) => {
+    app.use(`/${template}`, express.static(path.resolve(__dirname, '..', template, 'assets')));
```

Comment:
I wonder if it would be worth building a mini document tree walker to automatically just set each directories asset folder as available to remover the need to manually update the `templates` list as more tests are written. Will probably need some light error handling to ignore possible situations where an `assets` directory isn't found

Reply:
I definitely want the development server to automatically pull in templates and make them available for tests, very likely via the method you've outlined here. I'll take a swing at it in this PR, since it will pay off very soon for future tests.